deploy/terraform-datacommons-website/test/website_v1_test.go

```diff
@@ -0,0 +1,190 @@
+// Once certain states are removed, Terrafrom deletions will no longer destroy those resources.
+func removeStates(t *testing.T, options *terraform.Options) error {
+	args := []string{"state", "rm"}
+	terraform.RunTerraformCommand(t, options, append(args, "module.apikeys.google_apikeys_key.maps_api_key")...)
+	terraform.RunTerraformCommand(t, options, append(args, "module.esp.google_endpoints_service.mixer_endpoint")...)
+	terraform.RunTerraformCommand(t, options, append(args, "module.esp.google_project_service.project")...)
+	terraform.RunTerraformCommand(t, options, append(args, "google_compute_managed_ssl_certificate.dc_website_cert")...)
+	return nil
+}
+
+// veifyWebsiteDeployment tests the status of DC website deployment within k8s.
+func veifyWebsiteDeployment(t *testing.T) {
+	sleepBetweenRetries, err := time.ParseDuration("30s")
+	if err != nil {
+		t.Fatalf("Failed to parse sleep duration. Please fix the code in veifyWebsiteDeployment: %v", err)
+	}
+	options := http_helper.HttpGetOptions{
+		Url:       fmt.Sprintf("https://%s", WEBSITE_DOMAIN),
+		TlsConfig: &tls.Config{},
+		Timeout:   10, // Seconds
+	}
+	http_helper.HttpGetWithRetryWithCustomValidationWithOptions(
+		t,
+		options,
+		60, // Retries,
+		sleepBetweenRetries,
+		func(code int, _ string) bool {
+			// Validation function. For now, just check the status of the home page.
+			return code == http.StatusOK
+		},
+	)
+}
+
+// mixerPathTo returns relative path to mixer givin path within mixer repo.
+func mixerPathTo(relativePath string) string {
+	return filepath.Join("../../../../mixer", relativePath)
+}
+
+// testWebsiteDeployment is a helper function to test the actual website.
+// Website deployment is done in the context of k8s resources.
+// For integration testing, this function is to be sandwiched between
+// infra setup and teardown.
+func testWebsiteDeployment(t *testing.T) {
+	options := &helm.Options{
+		KubectlOptions: k8s.NewKubectlOptions(
+			"gke_custom-dc-test_us-central1-a_datacommons-us-central1-a", // context name
+			os.Getenv("KUBECONFIG"), // kubeconfig path
+			"website",               // namespace
+		),
+		SetValues: map[string]string{
+			"website.image.project":      IMAGE_PROJECT,
+			"website.image.tag":          RELEASE_WEBSITE_GITHASH,
+			"website.githash":            RELEASE_WEBSITE_GITHASH,
+			"mixer.image.project":        IMAGE_PROJECT,
+			"mixer.image.tag":            RELEASE_MIXER_GITHASH,
+			"mixer.githash":              RELEASE_MIXER_GITHASH,
+			"website.gcpProjectID":       PROJECT_ID,
+			"website.domain":             WEBSITE_DOMAIN,
+			"website.secretGCPProjectID": PROJECT_ID,
+			"mixer.gcpProjectID":         PROJECT_ID,
+			"mixer.serviceName":          fmt.Sprintf("website-esp.endpoints.%s.cloud.goog", PROJECT_ID),
+			"ingress.enabled":            "true",
+		},
+		SetFiles: map[string]string{
+			"mixer.schemaConfigs.\"base\\.mcf\"":   mixerPathTo("deploy/mapping/base.mcf"),
```

Comment:
you can use `` to avoid escape "

Reply:
I'm afraid that the consumer of this is the k8s server in go, which only takes double quotes.

Reply:
shouldn't they be complied and used?